pages/common/m4.md

```diff
@@ -0,0 +1,11 @@
+# m4
+
+> Macro processor.
+
+- Process macros in file:
```

Comment:
Perhaps this will read better?

```suggestion
- Process macros in a file:
```

Also, what is mean  by `process`? Does it execute them, or simply parse them?

Perhaps we can change `{{file}}` to `{{path/to/file}}`?